### ISSUE TITLE
Similar to `objectify`, to return `Map`

### ISSUE THREAD

**Issue by omid** [rayepps/radash#401]
Sometimes it's [recommended](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#objects_vs._maps) to use [`Map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map) instead of `Record`, and the keys are not limited to `number | string | symbol`.

So it makes sense to have another set of functions to use `Map`s.

**Comment by gperdomor** [comment]
a `mapify` function 🤔

**Comment by omid** [comment]
Yes, and if it is used in any other functions which accept or return `Record`

**Comment by Aryan-mor** [comment]
Yes, it will be good

**Comment by aleclarson** [comment]
Hey, @omid @Aryan-mor @gperdomor. This suggestion was implemented over at [radashi-org/radashi](https://github.com/radashi-org/radashi). It'd be great to have your support. You can use it today by installing `radashi@beta`. An official release is pending.
